uc_checkoutpayment/uc_checkoutpayment.module

```diff
@@ -108,6 +103,8 @@ function uc_checkoutpayment_variable_info() {
  * Implements hook_page_build().
  */
 function uc_checkoutpayment_page_alter(&$page) {
+  global $baseUrl;
```

Comment:
global $baseUrl is a self-defined constant.

We should use the drupal constant $base_url as described on https://api.drupal.org/api/drupal/developer%21globals.php/global/base_url/7.x